playwright-e2e/tests/brain/brain-regression-test.spec.ts

```diff
@@ -0,0 +1,370 @@
+import { test, expect } from '@playwright/test';
+import { setAuth0UserEmailVerified } from 'utils/api-utils';
+import { APP } from 'data/constants';
+import * as auth from 'authentication/auth-base';
+import { generateEmailAlias, generateUserName } from 'utils/faker-utils';
+import * as testutils from 'utils/test-utils';
+import * as email from 'utils/email-utils'
+import HomePage from 'pages/brain/home-page';
+import PrequalPage from 'pages/brain/prequal-page';
+import ResearchConsentPage from 'pages/brain/consent-page';
+
+const { BRAIN_USER_EMAIL, BRAIN_USER_PASSWORD, MIN_EMAIL_WAIT_TIME, BRAIN_BASE_URL, SITE_PASSWORD } = process.env;
+
+test('Brain statics', async ({page}) => {
```

Comment:
Forgot before, but not too late to mention `test.slow()`. For longer or slower tests without setting another timeout, https://playwright.dev/docs/api/class-test#test-slow-1